README.md

```diff
@@ -1,17 +1,23 @@
 
 # supported [![CI](https://github.com/stefanpenner/supported/workflows/CI/badge.svg)](https://github.com/stefanpenner/supported/actions/workflows/ci.yml)
 
-
-
-
 ## Usage
 
-
 ```sh
 npx supported <path/to/node_module>
 npx supported <[array/of/node_modules]>
 ```
 
+#### Optional Flags
+
+The `--date` flag enables a form of limited time travel, and attempts to run
```

Comment:
Should date be `--current-date`? I don't have a strong feeling here, but `date` is quite general. 🤷‍♂️

Reply:
Ya, we should brain storm some ideas for the name. Date is a decent MVP name, but doesn't feel quite right.

Reply:
Ya, I'm honestly not super opinionated about this, apart from the fact that `--date` is more ambiguous, and `--current-date` is slightly more specific to what it represents. My rule of thumb is generally "call it what it is", and not require folks to go to supporting documentation/usage to figure out how to use the flag.

Reply:
changed to `--current-date`